community-operators/dynatrace/dynatrace-monitoring.v0.2.0.clusterseviceversion.yaml

```diff
@@ -0,0 +1,215 @@
+    mediatype: image/png
+  install:
+    spec:
+      deployments:
+      - name: dynatrace-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: dynatrace-oneagent-operator
+          template:
+            metadata:
+              labels:
+                dynatrace: operator
+                name: dynatrace-oneagent-operator
+                operator: oneagent
+            spec:
+              containers:
+              - command:
+                - dynatrace-oneagent-operator
+                env:
+                - name: MY_POD_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
```

Comment:
This operator might not be configured to watch `AllNamespaces` What happens if you create a CR in a namespace other than `openshift-operators`? Does the oneagent operator do anything?

Reply:
As it stands, it certainly isn't configured to Watch All namespaces given that the value of `metadata.annotations['olm.targetNamespaces']` isn't provided to the operator.

Reply:
I have tried deploying it in a namespace other than `openshift-operators`, but the oneagent operator doesn't seem to create pods in that case.

Reply:
Great - update the `installModes` to only support `ownNamespace` and `singleNamespace`